### ISSUE TITLE
Can't `convox run` process types that are scaled to 0

### ISSUE THREAD

**Issue by mattmanning** [convox/rack#363]
**To reproduce:**

Take any running Convox app and scale one of its processes to 0, for example:

`convox scale worker --count 0`

Now try to `convox run worker ls`

**Expected result:**

Worker container is booted and `ls` is run in it.

**Actual result:**

`ERROR: no such process: worker`

**Comment by mattmanning** [comment]
@bobzoller in Slack: " I've got a variation on that issue.. `convox run` against a process that is scaled up but flapping (failing to start, retrying) gives `ERROR: cannot connect to Docker endpoint`
ends up working intermittently"

**Comment by nzoschke** [comment]
I am testing this here:

It's now working as expected:

```
$ convox scale
NAME  DESIRED  RUNNING  MEMORY
db    1        1        256   
web   2        4        256   
web2  0        0        256  

$ convox run web2 bash
root@ed6f08978922:/usr/local/apache2# 

$ convox ps
ID            NAME  RELEASE      SIZE  STARTED         COMMAND                       
f43cd7eca88f  db    RSSCMPCETRY  256   14 minutes ago  /docker-entrypoint.sh postgres
7437e39bf75d  web   RSSCMPCETRY  256   12 minutes ago  httpd-foreground              
f40c1993f081  web   RSSCMPCETRY  256   12 minutes ago  httpd-foreground              
ed6f08978922  web2  RSSCMPCETRY  0     9 seconds ago   sh -c bash       
```

**Comment by nzoschke** [comment]
This is fixed and released in 20160302193502
